addons/mail/static/src/models/welcome_view/welcome_view.js

```diff
@@ -42,6 +42,7 @@ function factory(dependencies) {
             if (this.discussPublicView.shouldAddGuestAsMemberOnJoin) {
                 await this.performRpcAddGuestAsMember();
             }
+            this.env.services.bus_service._restartPolling();
```

Comment:
only inside shouldAddGuestAsMemberOnJoin should be acceptable, if they are already member no need to restart